packages/web/src/hooks/useTabs/useTabs.tsx

```diff
@@ -1155,6 +1178,7 @@ const useTabs = ({
       onTabClick,
       shouldAnimate,
       isMobile,
+      isMobileV2,
```

Comment:
what is going on with all this isMobileV2 stuff

Reply:
the design for mobile tabs changed in these designs, so i made a new prop for this. Not great so I'll ask design about if we want to make all of the other ones conform to these styles